src/java/org/apache/cassandra/index/Index.java

```diff
@@ -557,27 +615,12 @@ default void validate(ReadCommand command) throws InvalidRequestException
      * @param rowFilter rowFilter of query to decide if it supports replica filtering protection or not
      * @return true if this index supports replica filtering protection, false otherwise
      */
+    //TODO Need to confirm whether SAI needs to implement this as false
     default boolean supportsReplicaFilteringProtection(RowFilter rowFilter)
```

Comment:
Is this TODO part of the porting or was it a future thing?

Reply:
@djatnieks This has a separate ticket STAR-183. The focus of this port is single node support. While multi-node queries will work there are a number of optimisations that we want to bring in separately once we have migrated our multi-node tests.